File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    let pos = 0; // 0,1,
    for(let i=0;i<nums.length;i++) {
        if(nums[i] > nums[pos]) {
            pos=pos+1;
            nums[pos] = nums[i];   
        }
    }
    return pos+1;
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cs
public class Solution {
    public int RemoveDuplicates(int[] nums) {
        int pos=0;
        for(var i=0;i<nums.Length;i++) {
            if(nums[i] > nums[pos]) {
                pos+=1;
                nums[pos] = nums[i];
            }
        }
        return pos+1;
    }
}